picoquictest/skip_frame_test.c

```diff
@@ -543,35 +543,20 @@ static int compare_lines(char const* b1, char const* b2)
 
 int picoquic_test_compare_files(char const* fname1, char const* fname2)
 {
-    FILE* F1 = NULL;
+    FILE* F1;
```

Comment:
I'd leave the default initialization to NULL. You never know. It's better to have the code to fail immediately than some random undefined behavior. The compiler will figure out that the default initialization is not necessary anyhow. With GCC that is the case at -O1 aleady, see here: https://godbolt.org/z/091yDs

Reply:
Not sure. That will also get you a warning, "value is set but never used".